rebasehelper/tests/test_output_tool.py

```diff
@@ -99,6 +168,19 @@ def test_text_output(self):
 
         with open(logfile) as f:
             lines = [y.strip() for y in f.readlines()]
-            assert lines == self.get_expected_output().split('\n')
+            assert lines == self.get_expected_text_output().split('\n')
+
+        os.unlink(logfile)
+
+    def test_json_output(self):
+        output = OutputTool('json')
+        data = self.get_data()
+
+        logfile = os.path.join(self.TESTS_DIR, REBASE_HELPER_RESULTS_LOG)
+        output.print_information(logfile)
+
+        with open(logfile) as f:
+            lines = [x.strip().replace("'", "") for x in f.readlines()]
+            assert '\n'.join(lines) == self.get_expected_json_output()
```

Comment:
I think it might be better to have the expected output in a dict, JSON from the log file could be loaded to another dict and then simple dict comparison could be made.